libs/amazonPayments.js

```diff
@@ -0,0 +1,59 @@
+var nconf = require('nconf');
+var amazonPayments = require('amazon-payments');
+var Bluebird = require('bluebird');
+
+// TODO better handling of errors
+
+var IS_PROD = nconf.get('NODE_ENV') === 'production';
+
+var amzPayment = amazonPayments.connect({
+  environment: amazonPayments.Environment[IS_PROD ? 'Production' : 'Sandbox'],
+  sellerId: nconf.get('AMAZON_PAYMENTS:SELLER_ID'),
```

Comment:
`AMAZON_PAYMENTS:SELLER_ID` doesn't exist in config.json, it uses `AMAZON_PAYMENTS_SELLER_ID `, same for other vars